src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -155,11 +155,13 @@ export interface MongoMedia extends MongoId {
   mediaId: string
   mediainfo?: {
     format?: {
-      duration?: number
+      duration?: string
     }
   }
 }
 
+const MILLISECONDS: number = 1000
```

Comment:
This could contain two milliseconds. Personally I don't think it is necessary having this in a constant. But otherwise it could say `ONE_SECOND_IN_MS` or `ONE_THOUSAND_MILLISECONDS` or `MILLISECONDS_TO_SECONDS_RATIO`.